manifests/master/generate_types.pp

```diff
@@ -4,6 +4,11 @@
 # @param enable
 #   Enable ``puppet generate types`` management
 #
+# @param monitor_type
```

Comment:
Line 1 description needs to be generalized since no longer using `incrond` exclusively.